### ISSUE TITLE
Better way to translate

### ISSUE THREAD

**Issue by yeltsin** [alimony/whatistranshumanism.org#6]
Hey,

I know it is made in Jekyll but I had to say... Is a pain to translate from English to Portuguese since everything is hard coded in HTML.

Maybe using Markdown? Since it can achieve the same results.

**Comment by alimony** [comment]
Yes, I've thought about that. It's been a while but as far as I can remember there were some issues with assigning arbitrary `id` attributes to elements to make anchor links work, and other HTML quirks that are a bit messy to do in Markdown.

But I do realize it's currently an obstacle for translation work, so until I've had time to investigate the Markdown option properly, if it would make it easier for you, you could just copy the text from the website into a document type of your choice and translate it there. Then just hand the document over to me and I'll do all the tedious/boring stuff with marking it up etc.

What do you think?

**Comment by yeltsin** [comment]
Hey,

I am making the boring stuff haha. Is an idea for the next version, maybe. :)

**Comment by alimony** [comment]
Cool, looking forward to it, just let me know if I can be of any assistance.
